lib/std/builtin.zig

```diff
@@ -52,6 +52,25 @@ pub const subsystem: ?SubSystem = blk: {
 pub const StackTrace = struct {
     index: usize,
     instruction_addresses: []usize,
+
+    pub fn format(
+        self: StackTrace,
+        comptime fmt: []const u8,
+        options: std.fmt.FormatOptions,
+        writer: anytype,
+    ) !void {
+        var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
+        defer arena.deinit();
+        const debug_info = std.debug.getSelfDebugInfo() catch |err| {
```

Comment:
Previously there wasn't anything about this structure that indicated it was a StackTrace for the current process (as opposed to a child process or something else).
Perhaps there should be an `enum { CurrentProcess, OtherProcess }`?

Alternatively, maybe we could save the buildid? Related: https://github.com/ziglang/zig/issues/3047